release-notes/schemas/dotnet-os-packages.json

```diff
@@ -0,0 +1,190 @@
+            "description": "Display name of nominal package.",
+            "type": "string"
+          },
+          "required-scenarios": {
+            "description": "Required scenarios for which the package must be used.",
+            "type": "array",
+            "items": {
+              "description": "Scenarios relating to package dependencies. \u0027All\u0027 includes both CoreCLR and NativeAOT while \u0027Runtime\u0027 is intended to cover CoreCLR, only.",
+              "enum": [
+                "all",
+                "runtime",
+                "https",
+                "cryptography",
+                "globalization",
+                "kerberos"
+              ]
+            }
+          },
+          "references": {
+            "description": "Related references.",
+            "type": [
+              "array",
+              "null"
+            ],
+            "items": {
+              "type": [
+                "string",
+                "null"
+              ]
+            }
+          }
+        },
+        "required": [
+          "id",
+          "name",
+          "required-scenarios",
+          "references"
+        ]
+      }
+    },
+    "distributions": {
+      "description": "Set of distributions where the product can be used.",
+      "type": "array",
+      "items": {
+        "description": "An operating system distribution, with required package install commands and specific packages for distribution releases.",
+        "type": [
+          "object",
+          "null"
+        ],
+        "properties": {
+          "name": {
+            "description": "Name of the distribution, matching ID in /etc/os-release.",
+            "type": "string"
+          },
+          "install-commands": {
+            "description": "Commands required to install packages within the distribution.",
+            "type": [
+              "array",
+              "null"
+            ],
+            "items": {
+              "description": "A command to be run to install packages",
+              "type": [
+                "object",
+                "null"
+              ],
+              "properties": {
+                "run-under-sudo": {
+                  "description": "Whether the command needs to be run under sudo.",
+                  "type": "boolean"
+                },
+                "command-root": {
+                  "description": "The command to be run, like apt.",
+                  "type": "string"
+                },
+                "command-parts": {
+                  "description": "The command parts or arguments that need to be used.",
+                  "type": "array",
+                  "items": {
+                    "type": [
+                      "string",
+                      "null"
+                    ]
+                  }
+                }
+              },
+              "required": [
+                "run-under-sudo",
+                "command-root",
+                "command-parts"
```

Comment:
It's possible that a command would not require any parts.

Reply:
Made it nullable. We'll see if that helps.